docs/docs/guides/with-expo.md

```diff
@@ -0,0 +1,48 @@
+---
+sidebar_position: 4
+---
+
+# Developing with Expo
+
+Expo is a popular development platform in the react-native ecosystem.
+
+## Custom Entrypoint
+
+By default, Expo points to an entrypoint in the `node_modules` package. The path for this is configured in the `"main"` field of `package.json`. To configure the Playback Service in this entrypoint file:
+
+1. Copy the file
```

Comment:
I have little to no experience with Expo, but this seems a little hacky, so my questions are:

1. Is this like an official thing that Expo says to do? And if so, is there documentation that we can link to?
2. Is this something that can be added as an enhancement to the rntp lib so that Expo folks don't have to go through it?

Reply:
@dctalbot cool, thanks for that context. I'd love to add that hyperlink directly to that Expo doc that you have in your comment.

Reply:
Just a note for this step: a lot of users are going to be using `expo-router` which uses a different entry point and will likely require modification of `app/_layout.js` instead.